tarynmartin/shopify-challenge
# Fix return data to go into useState hook

#### What’s this PR do?
- Fix app so that returned data is stored in useState hook

#### Where should the reviewer start?
- App.js for fixed hook and movies being passed into results component

#### How should this be manually tested?
- In terminal, run npm start
- In browser, open dev tools 
- Do a search and console.log movies to see what is returned
